utils.js

```diff
@@ -16,6 +16,16 @@ module.exports = {
         } else {
             res.send(401);
         }
-    }
+    },
     
+    /* Add entry to RSS table */
+    addRSS: function(connection, title, description, created_by) {
+
+        connection.query('INSERT INTO rss(title, description, author) VALUES(?,?,?)', [title, description, created_by], function (error, rows,field) {
```

Comment:
We might want to introduce a rate limiter or prevent duplicates (e.g., if someone hits the update link multiple times). Something like this (this is just pseudocode off the top of my head... you can probably come up with a better approach):
```node
if (recentlySeen.indexOf(title)) return; // ignore repeated entries
recentlySeen.push(title);
if (recentlySeen.length > 3) recentlySeen.shift();
```
A better approach might be something that prevents a duplicate for ~5 minutes or something like that.